src/kotlin/flwr/src/main/java/dev/flower/android/Worker.kt

```diff
@@ -0,0 +1,58 @@
+package dev.flower.android
+
+import android.content.Context
+import androidx.work.Constraints
+import androidx.work.CoroutineWorker
+import androidx.work.Data
+import androidx.work.ExistingPeriodicWorkPolicy
+import androidx.work.PeriodicWorkRequest
+import androidx.work.PeriodicWorkRequestBuilder
+import androidx.work.WorkManager
+import androidx.work.WorkerParameters
+import java.util.UUID
+import java.util.concurrent.TimeUnit
+
+class FlwrWorker(private val context: Context,
+                 workerParams: WorkerParameters,
+                 private val flwrClient: Client,
```

Comment:
In Python, we call it just `client`:
```suggestion
                 private val client: Client,
```